microsoft-edge/webview2/index.md

```diff
@@ -106,10 +105,10 @@ You are able to run WebView2 applications on the following versions of Windows.
 *   Windows Server 2016  
 *   Windows Server 2012  
 *   Windows Server 2012 R2  
-*   Windows Server 2008 R2**  
+*   Windows Server 2008 R2 \*\*  
 
 > [!IMPORTANT]
-> ** WebView2 support for Windows 7 and Windows Server 2008 R2 will follow the same support cycle as Microsoft Edge browser. Navigate to [here][EdgeOSSupport] for more details.   
+> \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 is planned during the same support cycle as Microsoft Edge browser.  For more information, navigate to [Microsoft Edge supported Operating Systems][DeployedgeMicrosoftEdgeSupportedOS].  
```

Comment:
> WebView2 support for Windows 7 and Windows Server 2008 R2 is planned during the same support cycle as Microsoft Edge browser.

@jm-trd-ms saying our support is planned DURING the same support cycle as Edge feels a little off. Maybe either the original sentence or `WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge browser`?

Reply:
Updated. 

```md
> [!IMPORTANT]
> \*\* WebView2 support for Windows 7 and Windows Server 2008 R2 has the same support cycle as Microsoft Edge.  For more information, navigate to [Microsoft Edge supported Operating Systems][DeployedgeMicrosoftEdgeSupportedOS].  
```